lightly/__init__.py

```diff
@@ -79,6 +79,11 @@
 
 
 import os
+from typing import Optional
+
+import packaging.version
+from packaging.version import InvalidVersion
+from packaging.version import Version as _Version
```

Comment:
Didn't want to import `from packing.version import Version` because it could be confusing to have a `lightly.Version` attribute.